### ISSUE TITLE
AdGuard service is not started on boot on Windows

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdGuardHome#2263]
Might be caused by this change: https://github.com/AdguardTeam/AdGuardHome/pull/2081
At least, this is the only relevant change that was done to the way how service is registered.

Please set up a Windows VM and check what went wrong with it.

Initially reported in the Telegram chat: https://t.me/adguard/99977

IMPORTANT: I am not sure there's a problem at all. The user claims that the service is launched normally when he starts the service. Maybe it's just some Windows-specific issue.

![image](https://user-images.githubusercontent.com/5947035/98109270-58ecf880-1eae-11eb-8c55-add24a7cb406.png)

**Comment by ameshkov** [comment]
In the user's case the issue went away as mysteriously as it appeared.

**Comment by Zelmanyr** [comment]
Hi,

I have this issue, I first put a shortcut of AdGuardHome.exe executable in shell:startup dependant on the user, it didn't work, at all. Next I put it in shell:common startup (all users) it also didn't work.. And then I set up a task with the task sheduler to start a program at boot, I selected AdGuardHome.exe and... it also didn't work. I honnestly don't know how to proceed... 

Thanks for your help.
